src/GitVersion.Core/Configuration/ConfigFileLocator.cs

```diff
@@ -39,7 +39,7 @@ public void Verify(string? workingDirectory, string? projectRootDirectory)
         return GetConfigFilePath(HasConfigFileAt(workingDirectory) ? workingDirectory : projectRootDirectory);
     }
 
-    public Config ReadConfig(string workingDirectory)
+    public GitVersionConfiguration ReadConfig(string workingDirectory)
```

Comment:
[`ConfigFileLocator`](https://github.com/GitTools/GitVersion/pull/3226/files#diff-6491018f0aac7f0c026e4566f551df102c3e2070267f4cfaebcedcf737604d9a) can probably be renamed to `ConfigurationFileLocator`.